foodsaving/conversations/api.py

```diff
@@ -55,6 +58,27 @@ def has_object_permission(self, request, view, message):
         return message.conversation.participants.filter(id=request.user.id).exists()
 
 
+class IsAuthorConversationMessage(BasePermission):
+    """Is the user the author of the message they wish to update?"""
+
+    message = _('You are not the author of this message')
+
+    def has_object_permission(self, request, view, message):
+        return request.user == message.author
+
+
+class IsWithinUpdatePeriod(BasePermission):
```

Comment:
Could you add one test where editing fails if the period is over? Similar to https://github.com/yunity/karrot-backend/blob/951c911eef639ee1ef43f7e7f8ae22ef106021c1/foodsaving/pickups/tests/test_feedback_api.py#L310